app/routes/admin.py

```diff
@@ -0,0 +1,11 @@
+from flask import request, redirect, url_for, abort, g
+from app.session.authorization import is_admin_json
+from app.helpers.render import render_error, render_json, render_template
+from app.server import server
+import app.controllers.admin as admin_controller
+
+
+@is_admin_json
+@server.route("/admin/duplicate_users")
+def duplicate_users():
+    return render_template('admin/duplicate_users.html', data=admin_controller.get_duplicate_users())
```

Comment:
I think adding an `is_admin_template` or something like that instead to return either `404` or `403` for admin pages rather than both JSON and HTML types for same endpoint which is awkward.